datafusion/sqllogictest/test_files/groupby.slt

```diff
@@ -3563,3 +3563,81 @@ logical_plan
 Sort: multiple_ordered_table.c ASC NULLS LAST
 --TableScan: multiple_ordered_table projection=[c]
 physical_plan CsvExec: file_groups={1 group: [[WORKSPACE_ROOT/datafusion/core/tests/data/window_2.csv]]}, projection=[c], output_ordering=[c@0 ASC NULLS LAST], has_header=true
+
+# Create an external table with primary key
```

Comment:
It would be good somewhere here to add an error test that unsupported sql returned an error (e.g. some sort of FULLTEXT contraint)

Reply:
Added a (negative) test case for foreign key.